book/src/getting_started.md

```diff
@@ -19,8 +19,11 @@ Rust compiler. Here are the system requirements (they're pretty modest):
 ## Setting Up
 
 > Note: This guide assumes you have nightly Rust and Cargo installed, and also
-> have a working Internet connection. Please take care of these prerequisites
-> first before proceeding.
+> have a working Internet connection. 
+> Please take care of these prerequisites first before proceeding.
+> See [rustup][ru] for an easy way to have two installations of rust.
```

Comment:
Now when I read this out loud, it doesn't sound that good, any suggestions for an edit?